components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/SCIMCommonConstants.java

```diff
@@ -112,5 +115,48 @@
     public static final String DOB_REG_EX_VALIDATION_DEFAULT_ERROR =
             "Date of Birth is not in the correct format of YYYY-MM-DD";
     public static final String DATE_OF_BIRTH_REGEX = "^\\d{4}-\\d{2}-\\d{2}$";
+
+    /**
+     * Enum which contains the error codes and corresponding error messages.
+     */
+    public enum ErrorMessages {
+
+        // SUO - SCIM User Operations.
+        INVALID_PASSWORD_RECOVERY_REQUEST("SUO-10000", "User attribute update is not allowed",
```

Comment:
Change the name INVALID_PASSWORD_RECOVERY_REQUEST

Reply:
Fixed https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/367/commits/7eb9fee089a75e7a984e91ef3ecb3c2ce5ee165a